src/assets/main-navigation.json

```diff
@@ -1,5 +1,5 @@
 {
-  "defaultHost": "http://localhost:5173",
+  "defaultHost": "https://topcoder.com",
```

Comment:
How can we specify the dev environment here?

Reply:
@testflyjets It will be up to each app to determine how it injects build vars.

You can see how I did it for platform ui here: https://github.com/topcoder-platform/platform-ui/pull/421/files

`src-ts/header/Header.tsx`

Reply:
I guess I don't understand how `main-navigation.json` file is used by uni-nav. Let's discuss this separately.